### PR TITLE
[BKNDLSS-26694]: Registration for push notifications on Android devices

### PR DESCRIPTION
**Pull request by JoeSilentJoe**

### CODE REVIEW

File: src/com/backendless/push/BackendlessFCMService.java [hunk 1]
```diff
@@ -146,7 +146,7 @@ private void fallBackMode( Context context, String message, String contentTitle,
     // android.os.Build.VERSION_CODES.O == 26
     if( android.os.Build.VERSION.SDK_INT > 25 )
     {
-      final String channelId = Backendless.getApplicationIdOrDomain() + ":" + channelName;
+      final String channelId = PushTemplateHelper.getChannelNotificationPrefix() + ":" + channelName;
```
Comment (oleg-vyalyh) [inline src/com/backendless/push/BackendlessFCMService.java:149]: Why did you remove appid ?  
What if the Android app works more that with one Backendless app ?

Comment (JoeSilentJoe) [reply src/com/backendless/push/BackendlessFCMService.java:149]: I did not delete the appid.
Not quite understood about _the Android app works more that with one Backendless app_, that is, in one application will be caused by _backendless.seturl()_ and _backendless.initapp ()_ to do something and again _backendless.seturl()_ and _backendless.initapp ()_ only with other arguments?
_Backendless.getApplicationidOrdomain()_ is used in 4 places as a prefix for composing _channelId_.
I did not renamed the _backendless.getApplicationIdOrdomain()_ method, and the method added
```
   Static Public String GetChannelNotificationPrefix ()
   {
     Return backendless.getApplicationIDORDOMAIN ();
   } 
```

**Review by oleg-vyalyh — APPROVED** [review]
